README.asciidoc

```diff
@@ -1,59 +1,108 @@
-Rudder: open source IT automation and compliance
-------------------------------------------------
+Rudder: Continuous configuration for effective compliance
+---------------------------------------------------------
 
-[horizontal]
-Rudder site:: http://rudder-project.org
-Source:: https://github.com/Normation/rudder
-User documentation:: https://www.rudder-project.org/site/documentation/user-manual/
-API documentation:: http://www.rudder-project.org/rudder-api-doc/
-Bug report:: http://www.rudder-project.org/redmine/projects/rudder/issues
-Mailing list, irc:: https://www.rudder-project.org/site/community/mailing-lists/
-
-Rudder is an easy to use, web-driven, role-based solution for IT Infrastructure Automation & Compliance.
+Rudder is an easy to use, web-driven, role-based solution for IT Infrastructure
+Automation & Compliance.
 
 The workflow offers different users options at their own level:
 
-*  Non-expert users can define parameters in a central console, and Rudder will automatically make sure that your IT services are installed, configured, running and in good health. All actions (checks, warnings, fixed errors…) are reported upon immediately in the user interface, keeping drift from nominal behaviour low.
-*  Expert users can configure how to implement these parameters on different systems, or build upon the pre-defined library provided.
+*  Non-expert users can define parameters in a central console, and Rudder will
+   automatically make sure that your IT services are installed, configured, running
+   and in good health. All actions (checks, warnings, fixed errors…) are reported upon
+   immediately in the user interface, keeping drift from nominal behaviour low.
+*  Expert users can configure how to implement these parameters on different systems,
+   or build upon the pre-defined library provided.
 *  Managers can consult compliance reports and access audit logs.
 
-Rudder combines the power of some of the best-of-breed open source tools in the field, adding reporting and higher level features to keep an eye on your IT: http://www.ncf.io/[the ncf framework] and http://cfengine.com/[CFEngine configuration management agent], http://www.fusioninventory.org/[the FusionInventory hardware and software inventory tool], reporting and higher level features to keep an eye on your information system. Rudder’s web interface is developed in Scala. 
-
+Rudder is an established project with several 10000s of node managed, in companies
+from small to biggest-on-their-field. Typical deployement manages 100 to 1000 nodes.
+The biggest known deployment in 2016 is 7000 nodes.
 
 TIP: Rudder is open source – we welcome ideas, comments, criticism, patches and any other contributions! So let's get a chat we our community https://www.rudder-project.org/site/support/chat-mailing-lists/[on irc]!
```

Comment:
This could probably be moved a bit after, after the feature overview for example.